docs/index.asciidoc

```diff
@@ -206,6 +206,15 @@ h| GeoIp
 
 |=======================================================================
 
+[id="plugins-{type}s-{plugin}-unsupported_ingest_processors"]
+==== Unsupported Ingest Processors
```

Comment:
review note: (I am okay to opt out this section) Luckily we don't have unsupported ingest pipelines yet. However, all managed ingest pipelines can be edited, it means there is still possibility to get runtime errors when unsupported processors involved in ingest pipeline.